src/main/options.js

```diff
@@ -48,6 +50,8 @@ function normalize(options, opts) {
   }
 
   rawOptions.astFormat = resolveParser(rawOptions).astFormat;
+  rawOptions.locEnd = resolveParser(rawOptions).locEnd;
```

Comment:
Rather than call `resolveParser` 3 times, could we store the return value as a local variable?